osu.Game/Localisation/CommonStrings.cs

```diff
@@ -99,6 +99,31 @@ public static class CommonStrings
         /// </summary>
         public static LocalisableString Description => new TranslatableString(getKey(@"description"), @"Description");
 
+        /// <summary>
+        /// "Always"
+        /// </summary>
+        public static LocalisableString Always => new TranslatableString(getKey(@"always"), @"Always");
+
+        /// <summary>
+        /// "Never"
+        /// </summary>
+        public static LocalisableString Never => new TranslatableString(getKey(@"never"), @"Never");
+
+        /// <summary>
+        /// "None"
+        /// </summary>
+        public static LocalisableString None => new TranslatableString(getKey(@"none"), @"None");
+
+        /// <summary>
+        /// "Full"
+        /// </summary>
+        public static LocalisableString Full => new TranslatableString(getKey(@"full"), @"Full");
+
+        /// <summary>
+        /// "Off"
+        /// </summary>
+        public static LocalisableString Off => new TranslatableString(getKey(@"off"), @"Off");
```

Comment:
I don't think this is going to work. These words are contextual and hard to translate in such a generic way in many languages (japanese as one example).